kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/adapters/KmFormItemAdapter.java

```diff
@@ -254,6 +299,27 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         }
     }
 
+    //Moves selected item to 1st position
+    private void filterDropdownList(List<KmFormPayloadModel.Options> dropdownList) {
```

Comment:
is there any simple way of moving to 1st position?

Reply:
A simpler way would be knowing the Index of the selected object beforehand. But to know the index, we need to iterate the list anyhow. The loop will now break if the selected value is found.

Reply:
if we call the function to remove it from the list. and then call function to add this to the 0 position?

Reply:
To remove the value also we need the index of the value.

Reply:
if equals and hashnode is implemented, direct remove function should work. although library will internally do the same processing but our code will be clean

Reply:
I might be missing something here. remove function on a list either needs the object to be removed or the index. We have neither beforehand.
Some things which I think of are:
1) Using Iterator: Same complexity with extra space for storing the removed value.
2) removeIf -> Will only work android N onwards
3) sorting the list -> Heavy